release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -594,6 +594,13 @@ Previously, creating an SR-IOV network required a cluster administrator to confi
 
 For more information, see xref:../networking/hardware_networks/configuring-namespaced-sriov-resources.html#configuring-namespaced-sriov-resources[Configuring namespaced SR-IOV resources].
 
+[id="ocp-4-20-unnumbered-bgp-peering_{context}"]
+==== Unnumbered BGP peering 
+
+With this release, {product-title} includes unnumbered BGP peering.
+This was previosuly available as a Technology Preview feature. 
+You can use the `spec.interface` field of the BGP peer custom resource to configure unnumbered BGP peering.
+
```

Comment:
Do we need to add xref:  For more information, see xref:../networking/ingress_load_balancing/metallb/metallb-configure-bgp-peers.adoc#nw-metallb-example-unnumbered-bgp-peering_configure-metallb-bgp-peers[Example: Specify BGP peers for unnumbered BGP peering]